pylint/checkers/base/comparison_checker.py

```diff
@@ -212,6 +217,18 @@ def _check_logical_tautology(self, node: nodes.Compare):
             suggestion = f"{left_operand} {operator} {right_operand}"
             self.add_message("comparison-with-itself", node=node, args=(suggestion,))
 
+    def _check_two_literals_being_compared(self, node: nodes.Compare):
+        """Check if two literals are being compared; this is always a logical tautology."""
+        left_operand = node.left
+        right_operand = node.ops[0][1]
+        operator = node.ops[0][0]
+        if not isinstance(left_operand, nodes.Const):
+            return
+        if not isinstance(right_operand, nodes.Const):
+            return
+        suggestion = f"{left_operand.value} {operator} {right_operand.value}"
+        self.add_message("comparison-of-two-literals", node=node, args=(suggestion,))
```

Comment:
```suggestion
        self.add_message("comparison-of-constants", node=node, args=(left_operand.value, operator, right_operand.value,))
```
We could say will always be True, or always be False, too. I wonder if it's worth it.

Reply:
>We could say will always be True, or always be False, too. I wonder if it's worth it.

We would need to evaluate the expression. And I feel that's better done by a separate and more general global subexpression eliminator. Is there such a thing in pylint already?